ca_qc_cote_saint_luc/people.py

```diff
@@ -1,66 +1,42 @@
 # coding: utf-8
 from utils import CanadianScraper, CanadianPerson as Person, CUSTOM_USER_AGENT
 
-import re
-from urllib.parse import urljoin
-
 COUNCIL_PAGE = 'https://cotesaintluc.org/city-government/council/#councilmembers'
 
 
 class CoteSaintLucPersonScraper(CanadianScraper):
     def scrape(self):
         page = self.lxmlize(COUNCIL_PAGE, user_agent=CUSTOM_USER_AGENT)
+        councillors = page.xpath('//section[contains(@class,"avia-team-member")]')[:-1]
 
-        mayor_url = page.xpath('//a[contains(text(), "Mayor")]/@href')[0]
-        mayor = self.scrape_mayor(mayor_url)
-        if mayor:
-            yield mayor
+        for councillor in councillors:
+            name = councillor.xpath('.//h3/text()')[0]
+            assert len(councillors), 'No councillors found'
 
-        councillors_url = page.xpath('//a[contains(text(), "Councillors")]/@href')[0]
-        cpage = self.lxmlize(councillors_url, user_agent=CUSTOM_USER_AGENT)
+            if (councillor.xpath('.//div[contains(@class,"team-member-job-title")][contains(.,"Mayor")]/text()')):
+                role = "Mayor"
+                district = "C\xf4te-Saint-Luc"
 
-        councillors = cpage.xpath('//tr[td//img]')[:-1]
+            else:
+                roles = councillor.xpath('.//div[contains(@class,"team-member-job-title")]/text()')
 
-        assert len(councillors), 'No councillors found'
-        for councillor_row in councillors:
-            img_cell, info_cell = tuple(councillor_row)
-            if info_cell.xpath('.//p//text()[contains(., "Vacant")]'):
-                continue
-            cells = [x.strip() for x in info_cell.xpath('.//text()') if re.sub('\xa0', ' ', x).strip()]
-            name = cells[0].replace('Councillor ', '')
-            district = info_cell.xpath('.//p[contains(text(), "District")]//text()')[0]
-            email = self.get_email(info_cell)
-            phone = self.get_phone(info_cell, area_codes=[438, 514], error=False)
-            img_url_rel = img_cell.xpath('.//img/@src')[0]
-            img_url = urljoin(councillors_url, img_url_rel)
+                for i in roles:
+                    sep = ','
+                    role = i.split(sep, 1)[0]
+                    district = i.split(sep, 1)[-1]
```

Comment:
Make similar changes to my comments in other PRs (move assert outside for-loop, use UTF-8 characters, don't repeat the XPath, add `[0]` instead of having a for-loop, etc. Once that's done, you can do `role, district = roles.split(',', 1)` which is shorter and clearer.